conf/app.routes

```diff
@@ -87,6 +87,9 @@ POST        /cases/:reference/change-appeal/:appealId           @uk.gov.hmrc.tar
 GET         /cases/:reference/sample/status                     @uk.gov.hmrc.tariffclassificationfrontend.controllers.SampleController.chooseStatus(reference: String)
 POST        /cases/:reference/sample/status                     @uk.gov.hmrc.tariffclassificationfrontend.controllers.SampleController.updateStatus(reference: String)
 
+GET         /cases/:reference/sample/returnStatus               @uk.gov.hmrc.tariffclassificationfrontend.controllers.SampleReturnController.chooseStatus(reference: String)
+POST        /cases/:reference/sample/returnStatus               @uk.gov.hmrc.tariffclassificationfrontend.controllers.SampleReturnController.updateStatus(reference: String)
```

Comment:
Should these endpoints have camelcase in? Looks like we typically hyphenate when multi word. Thought there was possibly some kind of accessibility concern around camel cased URLs?

Reply:
Another good spot.  Fixed

Reply:
in my defence, I got confused with the camelCase variable names (not same thing I know)